packages/slate/src/transforms/node.ts

```diff
@@ -716,7 +716,9 @@ export const NodeTransforms = {
 
       const rangeRef = Range.isRange(at) ? Editor.rangeRef(editor, at) : null
       const matches = Editor.nodes(editor, { at, match, mode, voids })
-      const pathRefs = Array.from(matches, ([, p]) => Editor.pathRef(editor, p))
+      const pathRefs = Array.from(matches, ([, p]) =>
+        Editor.pathRef(editor, p)
+      ).reverse()
```

Comment:
I'm not quite following this change? Maybe you want to make a call to range.isBackward and reverse if necessary ?

Reply:
thanks for preview. this issue is not related to range direction.

`unwrapNode` will call `liftNode` which does not support split mode when nested. so if we not reverse the order and call it from top to the bottom, it will remove all  blocks that wrap target node. so we reverse the order

Reply:
Thanks, that makes sense.